packages/dds/tree/src/core/schema-stored/schema.ts

```diff
@@ -105,7 +105,14 @@ export interface SchemaAndPolicy {
 }
 
 /**
- * Extra data needed to interpret schema.
+ * Extra data needed to interpret stored schema.
+ * @remarks
+ * This contains information which describes the semantics of things which can be referenced in stored schema.
```

Comment:
Nit (I personally find double "which" a bit hard to parse):
```suggestion
 * This contains information that describes the semantics of things which can be referenced in stored schema.
```